packages/dd-trace/src/opentracing/span.js

```diff
@@ -97,7 +97,10 @@ class DatadogSpan {
       unfinishedRegistry.register(this, operationName, this)
     }
     spanleak.addSpan(this, operationName)
-    startCh.publish(this)
+
+    if (startCh.hasSubscribers) {
+      startCh.publish({ span: this, fields })
```

Comment:
uuuuh, that's a major change or am i mistaken ?

Reply:
at the moment there is only one subscriber to `dd-trace:span:start` and it doesn't use arguments at all
https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/profiling/ssi-heuristics.js#L121

Reply:
why is this published then ? it's in the opentracing folder so maybe it's used by other people outside dd-trace ? worth checking

Reply:
@szegedi is the `dd-trace:span:start` channel for internal use only? 
We want to use it to add some tag to certain spans but we need some extra parameter, could there be a problem with the modification we are introducing in this PR?

Reply:
@szegedi can you give us your formal approval for this PR on this point ?